tests/Hedgehog.Tests/GenTests.fs

```diff
@@ -62,4 +63,16 @@ let ``dateTime shrinks to correct mid-value`` () =
         |> (fun x -> x.Split System.Environment.NewLine)
         |> Array.item 1
         |> System.DateTime.Parse
-    System.DateTime (2000, 1, 1) =! result
\ No newline at end of file
+    System.DateTime (2000, 1, 1) =! result
+
+[<Fact>]
+let ``uint64 doesn't return any out-of-range value`` () =
+    let gen = Gen.uint64 <| Range.constant 1UL UInt64.MaxValue
+    let actual = Gen.sample 0 100 gen
+    Assertions.test <@ actual |> List.contains 0UL |> not @>
```

Comment:
Do we need to qualify the module `Assertions`? `test <@ actual …` should work fine.